receiver/gitproviderreceiver/metadata.yaml

```diff
@@ -13,40 +13,78 @@ status:
 # this might need to be unique per sub receiver implementation
 resource_attributes:
   organization.name:
-      enabled: true
-      description: Git Organization or Project Name
-      type: string
+    enabled: true
+    description: Git Organization or Project Name
+    type: string
   git.vendor.name:
-      enabled: true
-      # github, gitlab, bitbucket, gittea
-      description: The name of the Git vendor/provider (ie. GitHub / GitLab)
-      type: string
+    enabled: true
+    # github, gitlab, bitbucket, gittea
+    description: The name of the Git vendor/provider (ie. GitHub / GitLab)
+    type: string
 
-## Attritbutes that will be uncommented when the rest of the metrics are added
+## Attributes that will be uncommented when the rest of the metrics are added
 attributes:
   repository.name:
-      description: The full name of the Git repository
-      type: string
+    description: The full name of a Git repository
```

Comment:
```suggestion
    description: The name of a Git repository
```

Reply:
This was just resolved, not sure how this one slipped through the cracks.